src/main/java/net/socialgamer/cah/customsets/CustomCardsService.java

```diff
@@ -70,7 +69,7 @@
    */
   private static final long VALID_SET_CACHE_LIFETIME = TimeUnit.MINUTES.toMillis(15);
 
-  private static final AtomicInteger cardIdCounter = new AtomicInteger(-(GameOptions.MAX_BLANK_CARD_LIMIT + 1));
+  private static final AtomicInteger cardIdCounter = new AtomicInteger(Integer.MIN_VALUE);
```

Comment:
I had to change this to count from the bottom up because the number of blank cards can be changed with the server running.